packages/language-support/src/antlr-grammar/CommandParser.g4

```diff
@@ -0,0 +1,28 @@
+parser grammar CommandParser;
+
+import CypherParser;
+
+options { tokenVocab = CommandLexer; }
+
+fullStatements: statementOrCommand (SEMICOLON statementOrCommand)* SEMICOLON? EOF;
+
+statementOrCommand: (statement | consoleCommand);
+
+consoleCommand: COLON (clearCmd | historyCmd | useCmd | paramsCmd);
+
+paramsCmd: PARAM paramsArgs?;
+
+paramsArgs: (CLEAR | listCompleteRule | map | lambda);
+
+lambda: unescapedSymbolicNameString EQ GT expression;
+
+clearCmd: CLEAR;
+
+historyCmd: HISTORY;
+
+useCmd: useCompletionRule symbolicAliasName?;
```

Comment:
Yes! But also because the semantics are different. USE in the database allows `USE graph.byName('myComposite.myConstituent')` which we don't allow in `:use`

Reply:
We also allow running it without passing an argument, which switches you to the default database